physical/postgresql/postgresql_test.go

```diff
@@ -32,15 +38,89 @@ func TestPostgreSQLBackend(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Failed to create new backend: %v", err)
 	}
+	pg := b.(*PostgreSQLBackend)
+
+	//Read postgres version to test basic connects works
+	var pgversion string
+	if err = pg.client.QueryRow("SELECT current_setting('server_version_num')").Scan(&pgversion); err != nil {
+		logger.Info(fmt.Sprintf("Failed to check for Postgres version: %v", err))
```

Comment:
It would be nice if this were at a Warn level.

Reply:
Actually, should the test fail here?

Reply:
It should fail, Ill fix it